infrastructure/index.ts

```diff
@@ -503,8 +562,11 @@ if (!pulumi.runtime.isDryRun()) {
 const aRecord = createAliasRecord(config.targetDomain, cdn);
 const aliasRecord = createAliasRecord(config.websiteDomain, cdn);
 
-export const archiveBucketUri = archiveBucket.bucket.apply(b => `s3://${b}`);
-export const websiteBucketUri = websiteBucket.bucket.apply(b => `s3://${b}`);
+export const archiveBucketUri = pulumi.interpolate`s3://${archiveBucket.bucket}`;
```

Comment:
I assume the archiveBucket and websiteBucket (and all related code) are part of what you meant by later cleanup?

Reply:
Yep, exactly.